_src/m2-sku-reindexing-form.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: m2-documentation
+title: SKU reindexing form
+permalink: /doc/m2/sku-reindexing-form/
+description: Learn about SKU reindexing form in the Algolia extension for Magento 2
+---
+
+## Access
+
+This feature can be accessed directly from the Magento2 back-office main menu :
+**Stores > Algolia Search > Reindex SKU(s)**
+
+## Purpose
+
+It might happens that for some reason, customers want to index one or many products to Algolia immediately, without waiting for the [the indexing queue](/magento/doc/m1/indexing-queue/).
```

Comment:
As far as I understood, we're talking about a M2 feature. This documentation links to a M1 page. Is it expected?